uint8 slave_number
string object_name
string value
---
string output
bool result
